terraso_backend/config/settings.py

```diff
@@ -139,3 +139,5 @@
     "JWT_REFRESH_EXP_DELTA_SECONDS", default="3600", cast=config.eval
 )
 JWT_ISS = config("JWT_ISS", default="https://terraso.org")
+
+PROFILE_IMAGES_S3_BUCKET = config("PROFILE_IMAGES_S3_BUCKET")
```

Comment:
I think it would be better to add the AWS credentials to the settings file as well. Let it be only on environment variables leaves it a bit over implicit. `django-storages` describes the settings here: https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings